_data/authors.yml

```diff
@@ -5,17 +5,24 @@ francois:
     web: https://twitter.com/baldassarifr
     image: /img/author/francois.jpg
     blurb: has worked on the embedded software teams at Sun, Pebble, and Oculus. He is currently the CEO of <a href='https://memfault.com'>Memfault</a>.
+    twitter: https://twitter.com/baldassarifr
+    linkedin: https://www.linkedin.com/in/fran%C3%A7ois-baldassari-63a8b91a/
+    github: https://github.com/franc0is
 chris:
     name: Chris Coleman
     email: chris@memfault.com
     image: /img/author/chris.jpg
     blurb: is the founder and CTO of Memfault. Prior to founding Memfault, Chris worked on the embedded software teams at Sun, Pebble, and Fitbit.
+    github: https://github.com/chrisc11
+    linkedin: https://www.linkedin.com/in/christopher-coleman-812aa06b/
```

Comment:
might assign web to this to. I noticed the other day my name link doesnt go anywhere

Reply:
Yeah, just have no idea which profile you want to advertise. Github or LinkedIn?